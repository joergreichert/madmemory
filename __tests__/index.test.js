/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import App from '../pages/index.js'


describe('With Snapshot Testing', () => {
  it('App shows "Menu"', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})