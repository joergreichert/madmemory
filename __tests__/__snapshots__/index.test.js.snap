// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Menu" 1`] = `
<div>
  <div
    className="jsx-3435870881"
  >
    <div
      className="jsx-3435870881 hero"
    >
      <h1
        className="jsx-3435870881 title"
      >
        MadMemory
      </h1>
      <p
        className="jsx-3435870881 description"
      >
        Can you remember it?
      </p>
      <div>
        <div
          className="jsx-307703491"
        >
          <div
            className="jsx-1888330655"
          >
            <div
              className="jsx-1888330655 entryRow"
            >
              <a
                className="jsx-307703491 card"
                href="./game/words"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                <h3
                  className="jsx-307703491"
                >
                  Spielen
                </h3>
              </a>
            </div>
          </div>
        </div>
        <div
          className="jsx-307703491"
        >
          <div
            className="jsx-1888330655"
          >
            <div
              className="jsx-1888330655 entryRow"
            >
              <a
                className="jsx-307703491 card"
                href="./settings"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                <h3
                  className="jsx-307703491"
                >
                  Einstellungen
                </h3>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
